apps/web/public/static/locales/en/common.json

```diff
@@ -1618,10 +1618,11 @@
   "delete_sso_configuration_confirmation": "Yes, delete {{connectionType}} configuration",
   "delete_sso_configuration_confirmation_description": "Are you sure you want to delete the {{connectionType}} configuration? Your team members who use {{connectionType}} login will no longer be able to access Cal.com.",
   "organizer_timezone": "Organizer timezone",
-  "email_no_user_cta": "Create your account",
+  "email_no_user_cta": "Join your team",
```

Comment:
@emrysal just want to confirm that if we make a change in existing key then other language translations are updated later or not?

Reply:
This is actually a good point - did we decide just to create new keys? 🤔

Reply:
i think @PeerRich mentioned once in some PR that we should create new keys if making some change